server/routes/index.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+const router = express.Router();
+
+// Homepage
+router.get('/', (req, res) => {
+    res.json({
+        message: 'success',
+        data: [],
+    });
+});
+
+// About page
+router.get('/about', (req, res) => {
+    res.json({
+        message: 'success',
+        data: [],
+    });
+});
```

Comment:
these changes look like changes I've seen in other PRs. Please fix this. the only changes that should appear in a PR are the changes that handle that PR's functionality.